source/developers/cook-books/graphql/index.rst

```diff
@@ -0,0 +1,202 @@
+
+When a change is made in the content model, for example adding a new field or creating a new content-type, the
+GraphQL schema will be rebuilt to reflect the same changes. So for a Crafter CMS site that uses GraphQL queries the
+develop process would look like this:
+
+1. Developers make changes in the content model
+2. Developers update the GraphQL queries to use the latest schema
+3. Authors make updates using the latest content model
+4. Publishers review & approve the author's work
+5. Publishers publish to live both the content model configuration & the content updates
+6. Crafter Deployer will handle the GraphQL schema rebuild in delivery
+
+You can also use the Crafter CMS GraphQL API from an external site or application, however in this case you will need to
+handle the schema reload using third party tools.
+
+--------------------------------
+Using GraphiQL in Crafter Studio
+--------------------------------
+
+GraphiQL is a simple GraphQL client that you can use in Crafter Studio to run GraphQL queries and explore the schema 
+documentation for a site without the need of any other tool. To access GraphiQL follow these steps:
+
+1. Login to Crafter Studio
+2. Click ``Dashboard`` next to the name of your site
+3. Click ``Site Config`` in the left sidebar
+4. Click ``GraphiQL`` in the left sidebar
+
+To explore the GraphQL schema you can click the ``Docs`` icon on the right side:
+
+.. image:: /_static/images/developer/graphql/graphiql.png
+        :width: 75%
+        :alt: GraphiQL
+        :align: center
+
+GraphiQL provides a convenient search navigation to quickly found a specific type or field:
+
+.. image:: /_static/images/developer/graphql/graphiql-doc.png
+        :width: 75%
+        :alt: GraphiQL Schema Documentation Explorer
+        :align: center
+
+To test GraphQL queries type them in the left text editor, GraphiQL will provide suggestions and validate the query
+against the schema in real time.
+
+.. image:: /_static/images/developer/graphql/graphiql-query.png
+        :width: 75%
+        :alt: GraphiQL Query Editor
+        :align: center
+
+----------------
+GraphQL Examples
+----------------
+
+Here you can find some examples on how to query content using GraphQL. The following examples use the built-in 
+``Website Editorial`` blueprint but the same concepts apply to any Crafter CMS site.
+
+For each content-type in the site you will find a field in the root Query, the name of the field is based on the
+name of the content-type so for ``/page/article`` the field will be ``page_article``.
+These fields contain two sub-fields, one is the ``total`` number of items found by the query and the other is a list
+of ``items``.
+
+.. note::
+  Because GraphQL only supports the underscore ``_`` character besides alphanumeric for names, if your content-type or 
+  field name contains a the dash ``-`` character it will be replaced with a double underscore ``__``. To avoid 
```

Comment:
`field name contains a the dash ``-`` character` -> `field name contains` **a** `dash ``-`` character`